crates/concrete_driver/tests/checks.rs

```diff
@@ -102,3 +102,36 @@ fn undeclared_var() {
         error
     );
 }
+
+#[test]
+fn call_count_mismatch() {
```

Comment:
Can you rename this to `call_param_count_mismatch` so it matches `CallParamCountMismatch`?